core/contracts/common/implementation/PermissionedExpandedERC20.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.5.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./MultiRole.sol";
+import "../interfaces/ExpandedIERC20.sol";
+
+/**
+ * @title An ERC20 with permissioned burning and minting. The contract deployer will initially 
+ * be the only minter and the owner who is capable of adding new roles. 
+ */
+contract PermissionedExpandedERC20 is ExpandedIERC20, ERC20, MultiRole {
+    enum Roles {
+        // Can set the minter and burner.
+        Owner,
+        // Addresses that can mint new tokens.
+        Minter,
+        // Addresses that can burn tokens that address owns.
+        Burner
+    }
+
+    constructor() public {
+        _createExclusiveRole(uint(Roles.Owner), uint(Roles.Owner), msg.sender);
+        address[] memory initialMinters = new address[](1);
+        initialMinters[0] = msg.sender;
```

Comment:
Why initialize with the caller as a minter? Do we use this default anywhere? It seems to me that the least opinionated default would be to initialize both the minter and burner as empty lists, and let the `Owner` add minters and burners after initialization.

Reply:
This will result in needing to do two more calls from the deployers. This is fine, however, especially given that most of the time a contract will be deploying this token.

Reply:
It is to replicate the behavior of the deprecated `ERC20Mintable` which sets the deployer as an original minter. It is assumed throughout the tests that the deployer can mint new tokens.

I can make changes across the tests however as it does make it less opinionated which is a good thing

Reply:
this is done